infra/blueprint-test/pkg/utils/version.go

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/go-testing-interface"
+	"golang.org/x/mod/semver"
+)
+
+// MinSemver validates gotSemver is not less than minSemver
+func MinSemver(t testing.TB, gotSemver string, minSemver string) error {
```

Comment:
nit: Since we are returning the error back, we can skip passing in the `t`. We usually pass in `t` if we want to error out the test like `t.Error()` within the func.
```suggestion
func MinSemver(gotSemver string, minSemver string) error {
```